Tools/wasm/build_wasi.sh

```
@@ -0,0 +1,37 @@
+#!/usr/bin/bash
+
+set -e -x
+
+if [ ! -f Modules/Setup.local ]; then
+    touch Modules/Setup.local
+fi
+
+# TODO: check if `make` and `pkgconfig` are installed
+
```

When I first ran this, I got:

```
Building Python out of the source tree (in /home/mdboom/Work/builds/cpython/builddir/build) requires a clean source tree (/home/mdboom/Work/builds/cpython/builddir/build/../..)
```

after the (somewhat) time-consuming configure step had already run.  Maybe detect this before hand to fail faster?

(But just a minor usability point).